modules/EvseManager/manifest.yaml

```diff
@@ -245,6 +245,11 @@ config:
       - X1
       - F
     default: X1
+  soft_over_current_timeout_ms:
+    description: >-
+      Allow for over current to be present for N ms in soft over current checking during AC charging.
+    type: integer
+    default: 7000
```

Comment:
A minimum value of 6000 is required here to not violate IEC61851-1

Reply:
I didn't want to change the current logic. It was hard-coded to 7000 before. I will change it since it was requested

Reply:
You can add `minimum: 6000` before or after `default: 7000` in the manifest.yaml

Reply:
All ready :)